OxfordBodleian/Juel-Jensen/BDLaethg35.xml

```diff
@@ -48,11 +153,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p><locus from="ir" to="iiv"/> and have been cut at the bottom.</p>
```

Comment:
"and have": is something is missing before "and"?

Reply:
no, it was a mistake, thanks!